pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -384,6 +405,136 @@ func (r *ReconcileComplianceScan) phaseLaunchingHandler(h scanTypeHandler, logge
 	return reconcile.Result{}, nil
 }
 
+// getRuntimeKubeletConfig gets the runtime kubeletconfig for a node
+// by fetching /api/v1/nodes/$nodeName/proxy/configz endpoint use the kubeClientset
+// and store it in a configmap for each node
+func (r *ReconcileComplianceScan) getRuntimeKubeletConfig(nodeName string) (string, error) {
+	// get the runtime kubeletconfig
+	kubeletConfigIO, err := r.ClientSet.CoreV1().RESTClient().Get().RequestURI("/api/v1/nodes/" + nodeName + "/proxy/configz").Stream(context.TODO())
+	if err != nil {
+		return "", err
+	}
+	defer kubeletConfigIO.Close()
+	kubeletConfig, err := ioutil.ReadAll(kubeletConfigIO)
+	if err != nil {
+		return "", err
```

Comment:
Similar comment here about logging. We failed to read the configuration for a node.